GoToProject/src/components/CustomMapView/index.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { FlexContainer, InputLabel } from '../styled';
+import MapView, {Marker, PROVIDER_GOOGLE} from 'react-native-maps';
+import {Dimensions} from 'react-native';
+
+
+const {width, height} = Dimensions.get('window');
+
+const ASPECT_RATIO = width / height;
+const LATITUDE_DELTA = 0.0922;
+const LONGITUDE_DELTA = 0.0421;
+
+const GOOGLE_MAPS_APIKEY = 'AIzaSyB4SwDlMm4Dr2c7j5A-iRbFbAWxIKO-tOA';
```

Comment:
You can call these constants from an external file.

Reply:
created a file for constants